src/CoreWCF.RabbitMQ/src/CoreWCF/Channels/RabbitMqTransportBindingElement.cs

```diff
@@ -12,26 +13,25 @@ namespace CoreWCF.Channels
 {
     public class RabbitMqTransportBindingElement : QueueBaseTransportBindingElement
     {
-        /// <summary>
-        /// Creates a new instance of the RabbitMQTransportBindingElement Class using the default protocol.
-        /// </summary>
         public RabbitMqTransportBindingElement()
         {
         }
-
+        
         private RabbitMqTransportBindingElement(RabbitMqTransportBindingElement other)
         {
```

Comment:
Should call the base constructor passing other as there might be some properties on QueueBaseTransportBindingElement or TransportBindingElement which need to be copied.